css/index.css

```diff
@@ -1,3 +1,250 @@
 /* Use your own code or past solution for Great Idea Web Page CSS here! */
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
 
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+
+body{
+	width: 900px;
+	height: 1400px;
+	margin: auto;
+}
+
+nav{
+	width: 900px;
+	display: flex;
+	justify-content: space-around;
+}
+.links{
+    display: flex;
+    justify-content: space-between;
+    width: 500px;
+}
+
+a{
+	text-decoration: none;
+	color: grey;
+}
+
+.topPart{
+	display: flex;
+	justify-content: space-around;
+}
+.button{
+    position: relative;
+    width: 200px;
+    top: -175px;
+    right: -110px;
+}
+button{
+    
+    background: white;
+    width: 150px;
+}
+
+h1{
+	height: 400px;
+	width: 400px;
+	font-size: 70px;
+	text-align: center;
+	padding: 75px;
+	margin-top: 75px;
+}
+
+.codePhoto{
+	padding: 75px;
+}
+
+.featuresAbout{
+    display: flex;
+    border-top: 1px solid black;
+    padding-top: 30px;
+    padding-bottom: 30px;
+}
+.bottom{
+    display: flex;
+    padding-top: 30px;
+    padding-bottom: 30px;
+    border-bottom: 1px solid black;
+}
+.contactPar{
+    padding-top: 30px;
+}
+footer{
+    padding-top: 30px;
+    display: flex;
+    justify-content: center;
+}
+/*media queries*/
+
+@media(max-width: 800px){
+    body{
```

Comment:
if percentages are used on a wrapper div to set the overall width of the page then you won't have to reset the width here.  It will automatically scale down.